web/forms.py

```
@@ -1869,3 +1873,53 @@ class StudyGroupForm(forms.ModelForm):
     class Meta:
         model = StudyGroup
         fields = ["name", "description", "course", "max_members", "is_private"]
+
+
+class VideoRequestForm(forms.ModelForm):
+    """Form for users to request educational videos on specific topics, with XSS protection."""
+
+    ALLOWED_TAGS: ClassVar[List[str]] = ["b", "i", "strong", "em", "ul", "ol", "li", "p", "a"]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)